config/locales/en/documents/show.yml

```diff
@@ -0,0 +1,51 @@
+en:
+  documents:
+    show:
+      title: No title
```

Comment:
Should this be `fallback_title` or something?

Reply:
I've renamed it - I was trying to be consistent with the other pages, but yeah it's not always the title.